bookstore/handlers.py

```diff
@@ -111,17 +118,14 @@ def load_jupyter_server_extension(nb_app):
     # Always enable the version handler
     web_app.add_handlers(host_pattern, [(base_bookstore_pattern, BookstoreVersionHandler)])
 
-    config = web_app.settings['config']
-    bookstore_settings = config.get("BookstoreSettings")
-
-    if not bookstore_settings:
+    if not nb_app.config.get("BookstoreSettings"):
         nb_app.log.info("Not enabling bookstore publishing since bookstore endpoint not configured")
     else:
         web_app.add_handlers(
             host_pattern,
             [
                 (
-                    url_path_join(base_bookstore_pattern, r"/published%s" % path_regex),
+                    url_path_join(base_bookstore_pattern, r"/publish%s" % path_regex),
```

Comment:
The reason I used the `published` name is that it's a noun -- it reflects that we have a published resource at `path`. `publish` reads like we're making an RPC call.

Reply:
Apologies! I misread your previous comment around making it more RESTy and thought that hadn’t been done yet, we can revert all the name changes with no issue (I was only making it because I thought it was something you had wanted but hadn’t gotten around to).